themes/LaserWave-color-theme.json

```diff
@@ -58,14 +61,30 @@
         "input.border": "#EB64B9",
         "inputOption.activeBorder": "#EB64B9",
 
-        "list.activeSelectionBackground": "#EB64B9CC",
-        "list.hoverBackground": "#EB64B9AA",
-        "list.hoverForeground": "#fff",
-        "list.inactiveSelectionBackground": "#EB64B9AA",
-        "list.inactiveSelectionForeground": "#fff",
+        "list.activeSelectionBackground": "#eb64b98f",
+        "list.activeSelectionForeground": "#eee",
+        "list.hoverBackground": "#EB64B98f",
```

Comment:
I would like to keep the hover background a bit more transparent so you can differentiate once you already have a selection

Reply:
Fixed in https://github.com/Jaredk3nt/laserwave/pull/19/commits/258e9ddcb26df8006fd586578dffbeb57a2a0f12